lib/get-file-paths.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path = require('path');
+
+/**
+ * Returns an object of file path data
+ * @function getFilePaths
+ * 
+ * @param	{object}	a vinyl file object
+ */
+exports.getFilePaths = function getFilePaths (file) {
```

Comment:
These function names are required. That's how the system exports this file's function. The pointer is in utility.js and removing the exports part causes an error.